src/types/dm_channel.rs

```diff
@@ -0,0 +1,17 @@
+#[derive(Deserialize, Clone)]
+pub struct DMChannel {
+    /// The channel ID
+    ///
+    /// 128 bit unsigned integer
+    pub id: u128,
+
+    /// The channel's name
+    /// Only applies to Group DMs
+    /// String of max length 100 chars
+    pub name: Option<String>,
+
+    /// The users inside the channel
+    ///
+    /// Vec of 128 bit unsigned integers
+    pub guild_id: Vec<u128>,
```

Comment:
why is this guild_id, if it is a vector? Shouldn't it be users?

Reply:
oh oops sorry lol my brain broke. will fix